### ISSUE TITLE
Policy Reporter: flag provided but not defined: -config

### ISSUE THREAD

**Issue by kravciak** [kubewarden/helm-charts#311]
When I enable `Policy Reporter` then `rancher-kubewarden-controller-ui` fails to start.
There is an error in the logs:

```
ε k get pods -n cattle-kubewarden-system
NAME                                                             READY   STATUS              RESTARTS       AGE
rancher-kubewarden-controller-policy-reporter-778c5554c4-vf6c5   1/1     Running             0              5m1s
rancher-kubewarden-controller-596f95ccb6-vfwhk                   1/1     Running             0              5m1s
audit-scanner-28266107-5trrk                                     0/1     ContainerCreating   0              4m39s
rancher-kubewarden-controller-ui-56cb789774-x52hf                0/1     CrashLoopBackOff    5 (115s ago)   5m1s

ε k logs -n cattle-kubewarden-system rancher-kubewarden-controller-ui-56cb789774-x52hf
flag provided but not defined: -config
Usage of /app/policyreporter-ui:
  -backend string
    	PolicyReporter Host
  -dev
    	Enable CORS Header for development
  -kyverno-plugin string
    	Kyverno Plugin Host
  -log-size int
    	Max amount of persistet results (default 200)
  -port int
    	PolicyReporter UI port (default 8080)
```

[Nightly test](https://github.com/rancher/kubewarden-ui/actions/runs/6346437441/job/17239933016) started to fail today on this error.

**Comment by kravciak** [comment]
With @viccuad we found out that in https://github.com/kubewarden/helm-charts/pull/294 we we downgraded kyverno/policy-reporter-ui from 1.8.4 (Published 5 months ago) to 1.15.1 (Published almost 2 years ago).

**Comment by viccuad** [comment]
The newly released `kubewarden-controller` chart `v1.6.2` contains a correct update of the policy-reporter dependency, and fixes this bug.

To not repeat this (or other) problems with policy-reporter chart dependencies in the future, I have opened:
- https://github.com/kubewarden/helm-charts/pull/314
- https://github.com/kubewarden/kubewarden-end-to-end-tests/issues/88
- https://github.com/kubewarden/helm-charts/issues/316
